operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -489,6 +492,69 @@ local utils = import 'mixin-utils/utils.libsonnet';
     type: 'text',
   } + options,
 
+  alertListPanel(title, nameFilter='', labelsFilter=''):: {
+    type: 'alertlist',
+    title: title,
+    options: {
+      maxItems: 100,
```

Comment:
This seems like a lot for an overview dashboard

Reply:
Good point. My rationale was that we shouldn't "silently" hide alerts, so I thought about putting an high number here. I don't feel strong and can reduce it. WDYT?

Reply:
OK, fine to leave it as is then.